### ISSUE TITLE
scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table, in Publish screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#883]
**Describe the bug**
scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table.

**To Reproduce**

1. Go to Publish  (http://localhost:9992/cm/app/?view=publish&site=test&path=%2FSites%2Ftest%2F&pathType=page)
2. Click on a Site.
3. On next screen 2 tables will be displayed i.e. Logs and Current status.
4. Inspect element and you will find that scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in tables.

**Expected behavior**
scope="col" for th tag and scope = "row" for very first td in each tr tag, in table, should be present.

**Additional context**
Navigate all the options available for similar issue and fix.
